bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/TopBNV/BNV_run_card.dat

```diff
@@ -0,0 +1,266 @@
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+10000 =  nevents  ! Number of unweighted events requested
+0	 =  iseed    ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+1	=  lpp1     ! beam 1 type
+1	=  lpp2     ! beam 2 type
+6500.0	=  ebeam1   ! beam 1 total energy in GeV
+6500.0	=  ebeam2   ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+0.0	=  polbeam1  ! beam polarization for beam 1
+0.0	=  polbeam2  ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+lhapdf = pdlabel      ! PDF set
+$DEFAULT_PDF_SETS = lhaid
+$DEFAULT_PDF_MEMBERS = reweight_PDF      ! if pdlabel=lhapdf, this is the lhapdf number
+#*********************************************************************
+# Renormalization and factorization scales                           *
+#*********************************************************************
+False	=  fixed_ren_scale   ! if .true. use fixed ren scale
+False	=  fixed_fac_scale   ! if .true. use fixed fac scale
+91.188	=  scale             ! fixed ren scale
+91.188	=  dsqrt_q2fact1     ! fixed fact scale for pdf1
+91.188	=  dsqrt_q2fact2     ! fixed fact scale for pdf2
+-1	=  dynamical_scale_choice  ! Choose one of the preselected dynamical choices
+1.0	=  scalefact         ! scale factor for event-by-event scales
+#*********************************************************************
+# Time of flight information. (-1 means not run)
+#*********************************************************************
+-1.0	=  time_of_flight  ! threshold below which info is not written
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+0	=  ickkw             ! 0 no matching, 1 MLM, 2 CKKW matching
+1	=  highestmult       ! for ickkw=2, highest mult group
+1	=  ktscheme          ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+1.0	=  alpsfact          ! scale factor for QCD emission vx
+False	=  chcluster         ! cluster only according to channel diag
+False	=  pdfwgt            ! for ickkw=1, perform pdf reweighting
```

Comment:
@agrohsje does changing this flag require reproducing the gridpacks?

Reply:
pdfwgt will only be relevant in the case of additional jets. reason why i changed was just to prevent a bug in case someone starts from this pr and adds jets. then it matters. at present this has no impact.